client/blocks/disconnect-jetpack-dialog/index.jsx

```diff
@@ -117,8 +125,62 @@ class DisconnectJetpackDialog extends PureComponent {
 		} );
 	}
 
+	disconnectJetpack = () => {
+		const {
+			onClose,
+			redirect,
+			site,
+			siteId,
+			siteTitle,
+			translate,
+			successNotice: showSuccessNotice,
+			errorNotice: showErrorNotice,
+			infoNotice: showInfoNotice,
+			removeNotice: removeInfoNotice,
+			disconnect: disconnectSite,
+			recordGoogleEvent: recordGAEvent,
+		} = this.props;
+
+		onClose();
+
+		recordGAEvent( 'Jetpack', 'Clicked To Confirm Disconnect Jetpack Dialog' );
+
+		const { notice } = showInfoNotice(
+			translate( 'Disconnecting %(siteName)s.', { args: { siteName: siteTitle } } ),
+			{
+				isPersistent: true,
+				showDismiss: false,
+			}
+		);
+
+		disconnectSite( siteId ).then(
+			() => {
+				// Removing the domain from a domain-only site results
+				// in the site being deleted entirely. We need to call
+				// `receiveDeletedSiteDeprecated` here because the site
```

Comment:
Just moved from elsewhere, I know, but this comment doesn't seem to quite match up with the code below, referring to `receiveDeletedSiteDeprecated`.